book/src/guides/connect-with-soju.md

```diff
@@ -2,19 +2,43 @@
 
 To connect with a [**soju**](https://soju.im/) bouncer, the configuration below can be used as a template. Simply change so it fits your credentials.
 
+```toml
+[servers.soju]
+nickname = "<your-nickname>"
+server = "<your-bouncer-url>"
+port = 6697
+[servers.soju.sasl.plain]
+username = "<your-username>"
+password = "<your-password>"
+chathistory = true
+```
+
 > 💡  as of 2025.1 Halloy supports [`chathistory`](../configuration/buffer.md#bufferchathistory), so the machine name (like `@desktop`) is no longer needed when `chathistory` is enabled
```

Comment:
it should link to: https://halloy.chat/configuration/servers.html#chathistory if we still want this notice.